src/evm/comptroller.ts

```diff
@@ -78,6 +78,9 @@ export const mainnets = [
   get(chains.mode.id, {
     [SABLIER_COMPTROLLER]: ["0x0000008ABbFf7a84a2fE09f9A9b74D3BC2072399", 29282069],
   }),
+  get(chains.monad.id, {
+    [SABLIER_COMPTROLLER]: ["0x0000008ABbFf7a84a2fE09f9A9b74D3BC2072399", 34574999],
```

Comment:
Should we define `0x0000008ABbFf7a84a2fE09f9A9b74D3BC2072399` in a constant at the top of the file?

Reply:
Can do.

Reply:
https://github.com/sablier-labs/sdk/pull/94/commits/bf60d13666bc1843089e04cff49055b515d1a3eb